dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/trackedentity/DefaultTrackedEntityAttributeService.java

```diff
@@ -277,11 +277,6 @@ else if ( ValueType.IMAGE == valueType )
         {
             return validateImage( value );
         }
-        else if ( trackedEntityAttribute.hasOptionSet() && !trackedEntityAttribute.isValidOptionValue( value ) )
```

Comment:
This code is used in the old importer. I don't know if it is safe to just remove it

Reply:
ok restored old code and added tests for option set in old tracker